x/cronos/keeper/precompiles/bank.go

```diff
@@ -18,7 +18,7 @@ import (
 
 const (
 	EVMDenomPrefix          = "evm/"
-	BankContractRequiredGas = 10000
+	BankContractRequiredGas = 21000
```

Comment:
Estimated to the gas needed for simple transfer (may be increased in the future)

Reply:
we should apply a consistent mythology to decides the precompile gases, for example do a simulated run of a typical call, collect the cosmos-sdk reported gas, multiply with a coefficient.

Reply:
Correct, If we want to use the cosmos cost as a ground basis, we should probably split all functions within the module precompiles to individual function precompiles (each function within the module could consumes a very different amount of gas) and for ibc we could also top up the cost of relaying (average costs per msgs)

But not sure we would have time to implement it in a short term, if we want to use a static cost then it would be better to bump the required gas to avoid DOS or attack on the relayer

Reply:
the relayer message, it make sense to estimate a gas table for each message type based on production numbers.

Reply:
> But not sure we would have time to implement it in a short term, if we want to use a static cost then it would be better to bump the required gas to avoid DOS or attack on the relayer

Agree, @yihuang what's your estimation to estimate the gas consumption based on Cosmos cost with coefficient.

If this is considered to be too complex at the moment. I agree we go for a static cost as a version 1, but this cost should be set higher enough to cover all cases but not average to avoid any DOS vector.